decentraland/unity-renderer
# feat: toast for not owning items in wearable collection

## What does this PR change?

Shows a toast when a player doesn't own any items from a Linked Wearable collection

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/toast-doesnt-own-item-wearble-collection
2. Open the backpack
3. Check that the toast is visible when selecting an item from the dropdown that you dont own any item of that collection.
4. Check that the toast closes when you click anywhere
5. Check that if you dont click anywhere, the toast fades out after 5 seconds

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
